manage_breast_screening/core/jinja2/components/check-in/macro.jinja

```diff
@@ -0,0 +1,3 @@
+{% macro check_in(presented_appointment, check_in_url, csrf_input) %}
+  {%- include 'components/check-in/template.jinja' -%}
```

Comment:
I just left a related comment on https://github.com/NHSDigital/dtos-manage-breast-screening/pull/636#discussion_r2478756416 about how this template/macro split seems unnecessary if it the component is specific to our service (e.g. if it takes a presenter as input rather than primitive types)

what do you think?